improver_tests/acceptance/test_estimate_emos_coefficients.py

```diff
@@ -185,8 +186,8 @@ def test_missing_dataset(tmp_path):
             "--cycletime", "20170605T0300Z",
             "--truth-attribute", "mosg__model_configuration=uk_det",
             "--output", output_path]
-    with pytest.raises(RuntimeError, match="Missing historical forecast"):
-        run_cli(args)
+    run_cli(args)
+    assert not os.path.isfile(output_path)
```

Comment:
Suggest using the pathlib functionality here eg. `assert not output_path.is_file()` to match with other pathlib usage in this code.

Reply:
Doh, yep!

Reply:
I've replaced the usage of `os` as suggested.

Reply:
:+1: